eureka/S5_lightcurve_fitting/s5_fit.py

```diff
@@ -0,0 +1,76 @@
+import numpy as np
+from ..lib import manageevent as me
+from ..lib import readECF as rd
+import glob, os
+from ..lib import sort_nicely as sn
+import matplotlib.pyplot as plt
+from importlib import reload
+
+def fitJWST(eventlabel, workdir, speclc_dir, fit_par, run_par_file, meta):
+    # load savefile
+    if meta == None:
+        meta = me.load(speclc_dir + '/S4_' + eventlabel + '_Meta_Save.dat')
+
+    #read in run params
+    import eureka.S5_lightcurve_fitting.parameters as p
+    reload(p)
+    run_par=p.Parameters(param_file=run_par_file)
+    # Create directories for Stage 4 processing
```

Comment:
This comment doesn't seem related to the code below, and it isn't clear to me that the code below would work in general. Where is the speclc folder made, and what .txt files are supposed to be in it? No such folder or text files are always made as an output from stage 4.

Reply:
This code is what Sebastian set up to run on the lightcurve files we were using before we had output from the full pipeline. So basically right now Stage 5 is disconnected from all the others and just works on this directory of files Sebastian set up. I think this part could get removed when we connect Stage 5 to Stage 4.